### ISSUE TITLE
Tooltip doesn't trigger on column series with cursor enabled unless set in template.tooltipText

### ISSUE THREAD

**Issue by xorspark** [amcharts/amcharts4#665]
The documentation states that when you have a chart cursor set, you can enable tooltips by setting `tooltipText` at the top level series property, otherwise you have to set it at the series' sub element template, e.g.

```
series.tooltipText = "..."; //cursor enabled
// ...
series.columns.template.tooltipText = ".."; //cursor is not enabled
```

This is true in column charts when you're using a category or date axis as the X axis, but not if you're using a value axis as an X axis.

Here's a [codepen](https://codepen.io/team/amcharts/pen/3a88f1a8d5c6bbf80ea19726ecd9c82f?editors=0010) with three charts. The first two charts are column charts with X value axes, however the top chart has `tooltipText` set at the top level and it does not generate a tooltip, but the middle one does. The bottom one is using a category axis with the tooltip set at the series level and that works as expected.

**Comment by martynasma** [comment]
Lemme explain why it is like it is.

When you have a `CategoryAxis` or a `DateAxis`, the axis has a very fine increments (categories, days, minutes, etc.) So whenever `XYCursor` is displayed, it looks for the closest category/date then displays series tooltips for any data items that fall into that category/date.

`ValueAxis` does not have fine increments - it's just a range of values, hence `XYCursor` not finding the nearest data items to display tooltip for, resulting in no tooltip at all, if you set `tooltipText` on series directly.

When you set it on a column, you get a tooltip because column is something solid to hover on.

Bottom line, it's not possible to generate tooltips via Cursor for all-value-axis charts.

That said, I'm not sure if those examples are purely hypothetical charts, or something close to the target functionality, but it looks like they could easily be replicated using `CategoryAxis`, effectively solving tooltip issue.

Hope this all makes sense :)

**Comment by martynasma** [comment]
I'm closing this, but please feel free to post any follow up questions you may have.
